src/support_code_library_builder/types.ts

```diff
@@ -25,12 +25,12 @@ export interface ITestStepHookParameter {
 
 export type TestCaseHookFunction<WorldType> = (
   this: WorldType,
-  arg?: ITestCaseHookParameter
+  arg: ITestCaseHookParameter
```

Comment:
This was a mistake from #1690. The argument isn't optional - we always pass it in - it's just up to the user-authored code whether it does anything with it.